src/accounts/channel/channel-account-handler.js

```diff
@@ -0,0 +1,25 @@
+const HttpClient = require('../../core/client/http-client');
+
+class ChannelAccountHandler {
+  constructor(teamConfService) {
+    this.teamConfService = teamConfService;
+  }
+
+  setDefault(teamId, channelId, alias) {
+    return HttpClient.validateAlias(this.teamConfService, teamId, alias)
+      .then(() =>
+        this.teamConfService.saveAccountForChannel(teamId, channelId, alias)
+      )
+      .then(() => true);
+  }
```

Comment:
why not just make this.teamConfService.saveAccountForChannel(teamId, channelId, alias) 
return true?